suite-native/fiat-rates/src/fiatRatesSelectors.ts

```diff
@@ -204,3 +212,55 @@ export const selectIsAccountWithRatesByKey = memoizeWithArgs(
     },
     { size: 100 },
 );
+
+export const selectEthereumTokenHasFiatRates = memoizeWithArgs(
```

Comment:
since you are returning only boolean and not doing any expensive operation memoization is uneccessary here

Reply:
fixed